lib/rubocop/cop/performance/times_map.rb

```diff
@@ -0,0 +1,39 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Performance
+      # This cop checks for .times.map calls.
+      # In most cases such calls can be replaced
+      # with an explicit array creation.
+      #
+      # @example
+      #
+      #   @bad
+      #   9.times.map do |i|
+      #     i.to_s
+      #   end
+      #
+      #   @good
+      #   Array.new(9) do |i|
+      #     i.to_s
+      #   end
+      class TimesMap < Cop
+        MSG = 'Use Array.new with a block instead of .times.map.'
```

Comment:
Use backticks to highlight the Ruby code (`Array.new` and `.times.map`).

(When we print out the offense messages, the backticks are removed and the highlighted area is shown in yellow.)